### ISSUE TITLE
Customize notification sound on android

### ISSUE THREAD

**Issue by rubo77** [vector-im/riot-android#997]
I see no option to choose a custom notification sound.

Root only uses the standard notification sound on my phone.

At least an option to change that global sound would be needed immediately 😃

**Comment by lukebarnard1** [comment]
This is especially annoying when the default (on e.g. Samsung S9) is a "ringing" as in one that continues for several seconds.

**Comment by huguesdk** [comment]
For me by default, the sound notifications were OK: I had the SMS notification sound for messages, and the ringtone for audio and video calls. The notification sound setting was set on the first choice (default notification sound).

Then I changed to a softer sound (because I had many message notifications). But apparently it was too soft as I was missing notifications (or maybe the sound didn’t play?). So I put back the default choice, but then it used the ringtone sound for messages. So I manually selected the same sound as for SMS, but now it uses it for both messages and calls.

Then I updated to the latest version (0.8.2) and now it stays stuck on that choice: I select another one and press OK but it doesn’t apply my choice.

I wish I could put back the default setting as it was before I did any change. Also, maybe a separate notification sound setting for calls could be useful?
